tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -270,11 +279,13 @@ compress_rootfs() {
 	# For some unobvious reason libc has executable bit set
 	# clean this up otherwise the find -executable will not work correctly
 	find . -type f -name "*.so.*" | while IFS= read -r file; do
+		file "${file}" | grep -q ELF || continue
```

Comment:
The devkit rootfs contains various files which either fail to be stripped at build time, or which turn out to be corrupted at runtime without the ELF file checks